### PR TITLE
2 quick fix for 2.0 ga test.

### PR DESCRIPTION
**Pull request by liuyu85cn**
1st rebuild job stop may core
2nd if gen empty subtask, task manager…

### CODE REVIEW

File: src/meta/processors/jobMan/RebuildJobExecutor.cpp [hunk 1]
```diff
@@ -64,6 +64,12 @@ meta::cpp2::ErrorCode RebuildJobExecutor::stop() {
 
     folly::collectAll(std::move(futures))
         .thenValue([] (const auto& tries) mutable {
+            if (std::any_of(tries.begin(), tries.end(), [](auto& t){
+                return t.hasException();
+            })) {
+                LOG(ERROR) << "RebuildJobExecutor::stop() RPC failure.";
+                return cpp2::ErrorCode::E_STOP_JOB_FAILURE;
+            }
```
Comment (panda-sheep) [inline src/meta/processors/jobMan/RebuildJobExecutor.cpp:72]: StatisJobExecutor::stop maybe also change

**Review by critical27 — APPROVED** [review]
🤣 Good job

**Review by panda-sheep — APPROVED** [review]
well done!

**Review by bright-starry-sky — APPROVED** [review]
Well done

**Pull request merged**